modules/cli/cli.go

```diff
@@ -52,6 +52,8 @@ func New(m metadata.Info) *CLI {
 
 	// webhook and validation options
 	flagSet.BoolVar(&deferCfg.ValidatingWebhookEnabled, "enable-validating-webhook", true, "Enable the validating webhook.")
+	flagSet.StringVar(&deferCfg.WebhookCertificateConfigBaseImage, "webhook-certificate-config-base-image", consts.WebhookCertificateConfigBaseImage, "The base image for the certgen Jobs.")
+	flagSet.StringVar(&deferCfg.WebhookCertificateConfigShellImage, "webhook-certificate-config-shell-image", "busybox", "The shell image for the certgen Jobs.")
```

Comment:
Those 2 flags should be bound to the `Config` struct instead of the defer config as there's nothing to evaluate here.

Reply:
Got it. I just misunderstood the `flagsForFurtherEvaluation`. I thought that those are the flags which are still evaluated to keep, modify or drop by the team. But it is clear that further evaluation means that flags which needs some extra logic and cannot be assigned directly to the configuration.

I will create unit tests and then push the changes.